bridge/bridge/api/static/openapi/openapi.yaml

```diff
@@ -0,0 +1,919 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /wrap/requests:
+    get:
+      summary: Returns wrap requests information.
+      description: |
+        Retrieves a paginated list of wrap requests detected on the native network.
+        These represent user-initiated deposits that trigger wrapping.
+      tags:
+        - Wrap
+      operationId: getWrapRequests
+      parameters:
+        - $ref: '#/components/parameters/Offset'
+        - $ref: '#/components/parameters/Limit'
+        - $ref: '#/components/parameters/Sort'
+        - $ref: '#/components/parameters/PayoutStatus'
+      responses:
+        200:
+          description: List of wrap requests recorded by the bridge.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/WrapRequest'
+        400:
+          description: Invalid request parameters (e.g., invalid pagination or filters).
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /wrap/requests/{address}:
+    get:
+      summary: Returns wrap requests filtered by sender or destination address.
+      description: |
+        Retrieves wrap requests associated with the provided address.
+        This includes both source (sender) and destination addresses.
+      tags:
+        - Wrap
+      operationId: getWrapRequestsByAddress
+      parameters:
+        - $ref: '#/components/parameters/Address'
+        - $ref: '#/components/parameters/Offset'
+        - $ref: '#/components/parameters/Limit'
+        - $ref: '#/components/parameters/Sort'
+        - $ref: '#/components/parameters/PayoutStatus'
+      responses:
+        200:
+          description: Wrap requests linked to the specified address.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/WrapRequest'
+        400:
+          description: The provided address is invalid for filtering wrap requests.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /wrap/requests/hash/{transaction_hash}:
+    get:
+      summary: Returns wrap requests by request or payout transaction hash.
+      description: Retrieves wrap requests matched by either the request hash or associated payout hash.
```

Comment:
You need to explain what are these hashes and how to obtain them.
_I guess_ one is the hash of the transaction where the initial deposit was made that initiated the request?
And the other one is the hash of the transaction on the wrapped network where the wrapped token was transferred to the user? Wouldn't the request be finished then?
As you can see, I am confused, and I shouldn't need to guess anything :)

Also, if the transactions happen on different networks they probably have different formats, so that's worth mentioning.

Also, if we're still talking about requests being processed you need to say it. You can use the word "ongoing".